src/components/sbb-checkbox/sbb-checkbox.scss

```diff
@@ -0,0 +1,94 @@
+:host {
```

Comment:
<img width="551" alt="Bildschirmfoto 2022-09-07 um 17 28 47" src="https://user-images.githubusercontent.com/29278232/188918112-af97c445-82d8-475c-a45c-1ad884e2a12e.png">
I asked @mcilurzo on how this should look in the case of lot of text. I will inform you, if I receive an answer. Anyways, please also provide a storybook story for a long text variant to cover it with chromatic regressions tests.